examples/gem/MODULE.bazel

```diff
@@ -0,0 +1,28 @@
+"Bazel dependencies"
+bazel_dep(name = "bazel_skylib", dev_dependency = True, version = "1.5.0")
+bazel_dep(name = "rules_ruby", dev_dependency = True, version = "0.0.0")
+
+local_path_override(
+    module_name = "rules_ruby",
+    path = "../..",
+)
+
+ruby = use_extension("@rules_ruby//ruby:extensions.bzl", "ruby")
+
+ruby.toolchain(name = "rules_ruby", version = "3.2.1")
```

Comment:
This should load the version from `ruby_version.bzl` until multiple toolchains are implemented.

Reply:
the MODULE.bazel file cannot `load()`. We can discuss alternative ways to make these line up.